main.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 const AutoRunFile = ".autorun"
-const Version = "1.22.4dev"
+const Version = "1.22.5dev"
```

Comment:
The version goes by the future released version + dev. So doing this, you just skipped a version.